app/views/user_mailer/invited_to_collection_notification.text.erb

```diff
@@ -1,13 +1,11 @@
 <% content_for :message do %>
-Dear <%= @user.default_pseud.byline %>,
+<%= t(".greeting", name: @user.default_pseud.byline) %>
 
-The collection maintainers of <%= @collection.title %> would like to include your work (<%= @work.title.html_safe %>) in their collection!
+<%= t(".part1", collection: @collection.title, work: @work.title.html_safe ) %>
 
-To approve or reject this request, please visit your "Collection Items" page: <%= user_collection_items_url(@user) %>.
+<%= t(".text.part2", collection_items_url: user_collection_items_url(@user)) %>
 
-If in future you would prefer to automatically approve requests to add your
-works to collections, please edit your Preferences (<%= "#{root_url}users/#{@user.default_pseud.name}/preferences" %>)
-and tick the box next to "Automatically agree to your work being collected by others in the Archive.".
+<%= t(".text.part3", link: user_preferences_url(@user)) %>
```

Comment:
Can we call this `url` or `preferences_url`? Something that'll make it clearer to Translation that it's a URL, not a hyperlink with text they need to tweak.

Reply:
Fixed